releasenotes/notes/add-default-value-to-registry-3a91e6b20f29f428.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    "Add a default value to the registry on Windows: C:\ProgramData\datadog"
```

Comment:
The release notes are meant for the users. We should explain the impact for them (ie: why we did this).

Reply:
Suggested:

`Adding a default location for the file storing pointers to make sure we never loose any logs.`